spec/lib/vanagon/platform/windows_spec.rb

```diff
@@ -18,9 +31,16 @@
     context "on #{plat[:name]} we should behave ourselves" do
       let(:platform) { plat }
       let(:cur_plat) { Vanagon::Platform::DSL.new(plat[:name]) }
+      let (:wixtestfiles) { "/tmp/spec/fixtures/wix/resources/windows/wix" }
 
       before do
         cur_plat.instance_eval(plat[:block])
+        FileUtils.mkdir_p(wixtestfiles)
+        FileUtils.cp_r("spec/fixtures/wix/resources/windows/wix", "#{wixtestfiles}/..")
+      end
+      after do
+        FileUtils.rm_rf("#{WORK_BASE}")
+        FileUtils.rm_rf("/tmp/spec")
       end
```

Comment:
Thats one I picked up from @mcasburn #269 ..
The ```instance_eval``` applies to all the tests (not just the file move ones).

I will see if I can change this one to either (:all) in which case it gets called once (I think), as we really only want to populate the wix test area once.